pubspec.lock

```diff
@@ -1004,10 +1004,10 @@ packages:
     dependency: transitive
```

Comment:
The change to this file doesn't look intentional. I think we don't want to update this as it is irrelevant to this commit.